components/CharCount.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import {css} from 'glamor'
+import { Label } from '@project-r/styleguide'
+
+import { formatLocale } from 'd3-format'
+import swissLocale from 'd3-format/locale/de-CH'
+
+const numberFormatter = formatLocale(swissLocale).format(',')
```

Comment:
use `lib/utils/format.js` (we generally separate with `&thinsp;` except in the content because we don't have number support yet)